assertj-core/src/test/java/org/assertj/core/internal/classes/Classes_assertIsPrimitive_Test.java

```diff
@@ -0,0 +1,105 @@
+package org.assertj.core.internal.classes;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.BDDAssertions.then;
+import static org.assertj.core.error.ShouldBePrimitive.shouldBePrimitive;
+import static org.assertj.core.test.TestData.someInfo;
+import static org.assertj.core.util.AssertionsUtil.expectAssertionError;
+import static org.assertj.core.util.FailureMessages.actualIsNull;
+
+import org.assertj.core.api.AssertionInfo;
+import org.assertj.core.internal.ClassesBaseTest;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests for
+ * <code>{@link org.assertj.core.internal.Classes#assertIsPrimitive(AssertionInfo, Class)}</code> .
+ *
+ * @author Manuel Gutierrez
+ */
+class Classes_assertIsPrimitive_Test extends ClassesBaseTest {
+
+  @Test
+  void should_pass_if_actual_has_byte_type() {
```

Comment:
use a parameterized tests similar to the one I suggested in `Classes_assertIsNotPrimitive_Test`